src/Raven.Studio/typescript/common/bindingHelpers/datePickerBindingHandler.ts

```diff
@@ -44,6 +44,8 @@ class datePickerBindingHandler {
         const date : moment.Moment = ko.unwrap(valueAccessor());
         if (date) {
             $(element).data("DateTimePicker").date(date);
```

Comment:
please simplify this code.

Reply:
done
`$(element).data("DateTimePicker").date(date || null);`